chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/SurveillanceController.java

```diff
@@ -668,4 +680,19 @@ private void checkUploadedSurveillanceOwnership(Surveillance pendingSurv) {
 			} 
 		}
 	}
+	
+	private void updateAuthority(Surveillance surv){
```

Comment:
Is this method intended to update the Authority of an already created Surveillance? If so, it should not do that. Once a Surveillance has been created with a particular ROLE, then that ROLE shall not be able to be changed